src/app/components/character-count/character-count.stories.ts

```diff
@@ -0,0 +1,53 @@
+import { Meta, moduleMetadata, Story } from "@storybook/angular";
+import { CommonModule } from "@angular/common";
+import { UsaCharacterCountDirective, UsaCharacterCountModule } from "@gsa-sam/ngx-uswds";
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+import { CharacterCountBasic } from "./character-count-basic/character-count-basic.component";
+import { generateConfig } from "src/app/shared/sandbox/sandbox-utils";
+
+const characterCountFooter = require('!!raw-loader!./character-count-footer.component.html');
+
+export default {
+  title: 'Components/CharacterCount',
+  decorators: [
+    moduleMetadata({
+      imports: [CommonModule, UsaCharacterCountModule, ReactiveFormsModule],
+      declarations: [CharacterCountBasic]
+    })
+  ],
+  args: {
+    usaCharacterCount: 25,
+  },
+} as Meta;
+
+export const Overview = (args) => ({
+  template: characterCountFooter.default
+});
+Overview.parameters = {options: {showPanel: false}};
+
+const basicTemplate = (args) => ({
+  template: `<character-count-basic [characterCount]="${args.usaCharacterCount}"></character-count-basic>`
+})
+
+export const Basic = basicTemplate.bind({});
+Basic.parameters = {
+  preview: generateConfig('components/character-count/character-count-basic', 'CharacterCountBasicModule', 'character-count-basuc')
```

Comment:
Noticed a typo here. It does not seem to break anything, but probably something to fix.

Reply:
Yes, good catch on the typo. The selector there is used in code sandbox - the main app component in the sandbox environment imports the module - `CharacterCountBasicModule` in this case, and then in the app component's template, drops the selector - `character-count-basic` in this case. So the typo would break the sandbox demo. Fixed